MediaBrowser.Controller/Entities/BaseItem.cs

```diff
@@ -2918,5 +2918,27 @@ public virtual long GetRunTimeTicksForPlayState()
         public static readonly IReadOnlyCollection<ExtraType> DisplayExtraTypes = new[] { Model.Entities.ExtraType.BehindTheScenes, Model.Entities.ExtraType.Clip, Model.Entities.ExtraType.DeletedScene, Model.Entities.ExtraType.Interview, Model.Entities.ExtraType.Sample, Model.Entities.ExtraType.Scene };
 
         public virtual bool SupportsExternalTransfer => false;
+
+        public override bool Equals(object obj)
+        {
+            Logger.LogDebug("Comparing me ({0}) to generic them ({1})", this, obj);
+            return this.Equals(obj as BaseItem);
```

Comment:
Not 100% sure, but you might get marginally better performance doing `return obj is BaseItem baseItem && Equals(baseItem);` Personally I find that a bit more readable as well.

Reply:
Only diff in perf would be 1 less function call. Doing `is` or using `as` and checking for `null` is practically the same

Reply:
I don't believe in much perf difference here (and especially I don't believe it matters a lot in this place), but it certainly is more readable. Thanks!